crypto/vrf/vrf_r2ishiguro_test.go

```diff
@@ -0,0 +1,24 @@
+//go:build !libsodium && !coniks
+// +build !libsodium,!coniks
+
+package vrf
+
+import (
+	"crypto/ed25519"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestProveAndVerify_r2ishiguro_by_crypto_ed25519(t *testing.T) {
```

Comment:
Why is this the only snake case?

Reply:
I can only say that I was in the mood at that time.

Reply:
I feel that it's easy to look at the result on the display.

Reply:
Anyway, I'll change the function names with the camel case.

Reply:
I also understand and agree that the snake case look is better in some cases, such as describing a combination of patterns, as in this case.